metaflow/plugins/kfp/kfp.py

```diff
@@ -535,6 +552,64 @@ def _set_container_resources(
             container_op.container.set_ephemeral_storage_limit(
                 resource_requirements["local_storage_limit"]
             )
+        if (
+            kfp_component.pytorch_distributed_decorator
+            or "volume" in resource_requirements
```

Comment:
I'm wondering if we make the arguments `volume` or `volumes` so users could mount many volumes. I wonder as well about making the `volume` argument accept a `str` (ie. just the size) or a `dict`/`object` type (ie. some complex "Volume" type that can accept the `volume_dir`, `size`, `mode`, etc..

Obviously can be added later as I think this would be possible to make backwards compatible.